panther_docking/test/test_panther_charging_dock.cpp

```diff
@@ -149,38 +135,6 @@ void TestPantherChargingDock::ConfigureAndActivateDock(double panther_version)
   charging_dock_->activate();
 }
 
-void TestPantherChargingDock::CreateChargerEnableService(bool success)
-{
-  charger_enable_service_ = node_->create_service<std_srvs::srv::SetBool>(
-    "hardware/charger_enable", [success, this](
-                                 const std::shared_ptr<std_srvs::srv::SetBool::Request> request,
-                                 std::shared_ptr<std_srvs::srv::SetBool::Response> response) {
-      response->success = success;
-      if (success) {
-        this->charging_status_ = request->data;
-      } else {
-        this->charging_status_ = false;
-      }
-
-      if (this->charger_state_pub_) {
-        PublishChargerStatus(this->charging_status_);
-      }
-    });
-}
-
-void TestPantherChargingDock::CreateChargerStatusPublisher()
-{
-  charger_state_pub_ = node_->create_publisher<panther_msgs::msg::ChargingStatus>(
-    "battery/charging_status", rclcpp::QoS(1));
-}
-
-void TestPantherChargingDock::PublishChargerStatus(bool charging)
-{
-  panther_msgs::msg::ChargingStatus msg;
-  msg.charging = charging;
-  charger_state_pub_->publish(msg);
-}
-
 void TestPantherChargingDock::SetBaseLinkToOdomTransform()
```

Comment:
Why don't we create one method for setting transform accepting parent and child frame with stamp and transform as arguments? So we can use it for all scenarios? The transform values and frame names can be allocated once globally o we can easily refer to them in the test cases stage.